### PR TITLE
Config a list to define which test must be done by a specific browser

### PR DESCRIPTION
**Pull request by alcapat**
Use different class instead of big one for checker_test. 
Setup a list of tests by browser (more easier to read)

### CODE REVIEW

File: test/saucelabs/test.py [hunk 1]
```diff
@@ -155,35 +128,28 @@ def parse_args(args):
...
+                command_executor='http://' +
+                saucelabs_user +
+                ':' +
+                saucelabs_key +
+                '@ondemand.saucelabs.com:80/wd/hub',
+                desired_capabilities=current_desired_cap)
+            driver.implicitly_wait(DEFAULT_WAIT_FOUND)
+
+            if driver.name == "MicrosoftEdge":
+                print 'Force set version, strange...'
```
Comment (loicgasser) [inline test/saucelabs/test.py]: what is the reason for that?

Comment (loicgasser) [inline test/saucelabs/test.py]: meaning you define a certain version with IE edge when creating the driver and the version gets changed?

File: test/saucelabs/test.py [hunk 2]
```diff
@@ -56,29 +56,6 @@ def parse_args(args):
     top_browser = {'platform': "Windows 7", 'browserName': "firefox",
```
Comment (loicgasser) [inline test/saucelabs/test.py]: This should probably be part of `desired_cap_list` as well no?

File: test/saucelabs/checker_test.py [hunk 3]
```diff
@@ -81,222 +107,217 @@ def wait_url_changed(driver, old_url, timeout=DEFAULT_WAIT):
...
-        print "  Test python Translations"
-        driver.get(url_4_api + 'testi18n?lang=de')
-        assert "BLN" in driver.page_source
-        assert "Jagdbanngebiete" in driver.page_source
-        driver.get(url_4_api + 'testi18n?lang=en')
-        assert "ILNM" in driver.page_source
-        assert "Swiss game reserves" in driver.page_source
-        driver.get(url_4_api + 'testi18n?lang=toto')
+
+def ShortenUrl(driver, url, url_4_api):
```
Comment (loicgasser) [inline test/saucelabs/checker_test.py]: minor: avoid cap for the first letter of a function

**Comment by loicgasser** [comment]
Some small comments otherwise I I like it. It's easy to configure tests for each browser. Great!

File: test/saucelabs/test.py (hunk 1 above)
Comment (alcapat) [inline test/saucelabs/test.py]: I don't know why but with Edge you loose the 'version' information. 
So i force to rewrite it, not so nice, i'm agree, if you have an other solution ?

File: test/saucelabs/test.py (hunk 2 above)
Comment (alcapat) [inline test/saucelabs/test.py]: we add the "top_browser" in the desired_cap_list

**Comment by gjn** [comment]
@alcapat Thanks alot for the improvements.

**Pull request merged**